### ISSUE TITLE
Feature: Expanded image view

### ISSUE THREAD

**Issue by dnisbetjones** [Kalissaac/Eigen#4]
Eigen loads in images nicely but there's no interaction available to expand to view or interact with any context menu. It would be great to get expanded views of images.

**Comment by Kalissaac** [comment]
I agree, what options would you hope to have in the context menu? Right now I'm thinking of "copy" and "save to disk"

**Comment by dnisbetjones** [comment]
Yeah, I would say copy and save are really all you need for the time being
